indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -482,12 +502,13 @@ sealed class Asn1Structure(
             postfix = "\n" + (" " * indent) + "}"
         ) { it.prettyPrint(indent + 2) }
 
-    override fun contentToString(): String = (if (isSorted) " SORTED " else "") + ", children=$children"
+    override fun contentToString(): String =
+        (if (shouldBeSorted) " should be SORTED " else "") + (if (isActuallySorted) " (is acutely SORTED) " else " (is NOT SORTED) ") + ", children=$children"
```

Comment:
![image](https://github.com/user-attachments/assets/bb315cef-1107-4805-a40a-9e2901c6b7cb)

i assume this should also be "actually"

Reply:
i would also shorten this; maybe
```
when {
    shouldBeSorted && isActuallySorted -> "SORTED"
    shouldBeSorted && !isActuallySorted -> "NON-COMPLIANT (UNSORTED)"
    else -> ""
```